_posts/2019-09-11-#16688.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: pr
 title: "Add validation interface logging"
-components: [logging]
+components: ["utils/log/libs"]
```

Comment:
["utils/log/libs", "validation"]

Reply:
This PR doesn't change anything in validation.